### ISSUE TITLE
redesign about us page 

### ISSUE THREAD

**Issue by Ayan-thecodeking** [Coders-Evoke-Community/CodersEvoke_website#200]
![aboutus](https://user-images.githubusercontent.com/78692566/129787232-37a34374-becc-4201-b3b2-3d8ba9dad817.jpeg)

**Comment by nirban256** [comment]
hey @Ayan-thecodeking can I work in it ?

**Comment by Ayan-thecodeking** [comment]
> hey @Ayan-thecodeking can I work in it ?

ya sure @nirban256 go ahead ....first make it for mobile view so that it will look responsiveness

**Comment by Ayan-thecodeking** [comment]
> > hey @Ayan-thecodeking can I work in it ?
> 
> ya sure @nirban256 go ahead ....first make it for mobile view so that it will look responsiveness

we'll  send  illustration soon

**Comment by nirban256** [comment]
ok, cool!

**Comment by Ayan-thecodeking** [comment]
>  illustrations are updated @nirban256 check in assests

**Comment by Ayan-thecodeking** [comment]
@nirban256 ARE U WORKING on this ??

**Comment by Ayan-thecodeking** [comment]
@RiyaGupta89 let me know ??

**Comment by RiyaGupta89** [comment]
@Ayan-thecodeking  Yes, I would like to work on this.

**Comment by Ayan-thecodeking** [comment]
Yup sure go ahead @RiyaGupta89 plz make it for mobile view to look more responsive

**Comment by RiyaGupta89** [comment]
@Ayan-thecodeking Is the codebase using bootstrap?

**Comment by RiyaGupta89** [comment]
@Ayan-thecodeking Should I use raw css or bootstrap. It is more comfortable for me to use raw CSS. But I can use bootstrap as well as you say.

**Comment by Ayan-thecodeking** [comment]
@RiyaGupta89 
 let me know about the updates
